### PR TITLE
Changed sponsorship.pdf, updated jobfair email.

### PR DESCRIPTION
**Pull request by zmmille2**
Slight changes. Large change to the sponsorship pdf

### CODE REVIEW

File: liquid/templates/conference/emails/jobfair_invite.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-We've revamped what sponsorship will look like this year, with three separate tiers to get involved with. We're offering companies more than ever before, and there are some fantastic new ways to get more out of Reflections | Projections. To learn more about our tiered sponsorship, see the following pdf (http://acm.illinois.edu/static/sponsorship.pdf), which breaks down the tiers into more specific detail.
+We've revamped what attendance will look like this year form the ground up. Notably, attendance is only possible through sponsorship in our new tiered sponsorship model. We're offering companies more ways to get involved, and there are some fantastic new ways to get more out of Reflections | Projections. To learn more about different sponsorship opportunities, see the following pdf (http://acm.illinois.edu/static/RP_2015_Sponsorship_Opportunities.pdf), which breaks down the tiers into more specific detail.
```
Comment (ace-n) [inline liquid/templates/conference/emails/jobfair_invite.txt]: form -> from

File: liquid/templates/conference/emails/jobfair_invite.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 The lowest tier level, Sapphire, will look almost the same as getting a spot at the job fair in the past. The price is the same as years past, at $1250.
```
Comment (ace-n) [inline liquid/templates/conference/emails/jobfair_invite.txt]: Perhaps explain what "almost" means - i.e. what's different?

File: liquid/templates/conference/emails/jobfair_invite.txt [hunk 3]
```diff
@@ -14,7 +14,7 @@ The Sapphire sponsorship tier includes the following:
  - Advertising for {{ company.company_name }} on the Reflections | Projections website
  - One-Year sponsorship to ACM@UIUC at the Bronze Sponsor level
 
-If you're interested in sponsoring at a higher level than Sapphire, please reach out to us at corporate@acm.uiuc.edu.
```
Comment (ace-n) [inline liquid/templates/conference/emails/jobfair_invite.txt]: Might remove "than Sapphire?" (Just a nit.)

File: liquid/templates/conference/emails/jobfair_invite.txt [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-We're excited to extend an invitation to {{ company.company_name }} to attend this year's annual Reflections | Projections job fair. The job fair will be held on Friday, October 2nd, 2015. Reflections | Projections is one of the largest student run conferences in the US, with over 800 students coming every year to the job fair, and we'd love to have {{ company.company_name }} involved.
+We're excited to extend an invitation to {{ company.company_name }} to attend this year's annual Reflections | Projections job fair. The job fair will be held on Friday, October 2nd, 2015. Reflections | Projections is one of the largest student run conferences in the US, with over 1,200 students coming every year to the job fair, and we'd love to have {{ company.company_name }} involved.
 
-We've revamped what sponsorship will look like this year, with three separate tiers to get involved with. We're offering companies more than ever before, and there are some fantastic new ways to get more out of Reflections | Projections. To learn more about our tiered sponsorship, see the following pdf (http://acm.illinois.edu/static/sponsorship.pdf), which breaks down the tiers into more specific detail.
```
Comment (ace-n) [inline liquid/templates/conference/emails/jobfair_invite.txt]: Minor nit: I'd remove ", and there are some fantastic new ways to get more out of Reflections | Projections"

File: liquid/templates/conference/emails/jobfair_invite.txt (hunk 3 above)
Comment (zmmille2) [inline liquid/templates/conference/emails/jobfair_invite.txt]: updated

File: liquid/templates/conference/emails/jobfair_invite.txt (hunk 4 above)
Comment (zmmille2) [inline liquid/templates/conference/emails/jobfair_invite.txt]: updated

**Comment by ace-n** [comment]
Merging since neither me nor Zach can test locally. Make sure to thoroughly test this before using it for anything important.

TL;DR: "I don't always test in prod, but when I do it's because I can't do it locally."
